packages/base/room.gts

```diff
@@ -246,6 +247,14 @@ class PatchField extends FieldDef {
   @field payload = contains(PatchObjectField);
 }
 
+// A map from card document hash to the first card fragment event id.
+// This map can be used to avoid sending the same version of the card more than once in a conversation.
+// We can reuse exisiting eventId if user attached the same version of the card.
+const cardDocToEventId: Map<string, string> = new Map();
```

Comment:
let' rename to `cardHashes`

Reply:
having this in module scope may collide with the work that @jurgenwerk is doing around the loader refactoring. just a heads up Matic on this...